tfhe/src/integer/server_key/radix_parallel/scalar_div_mod.rs

```diff
@@ -220,6 +220,12 @@ impl ServerKey {
         // Rust has a check on all division, so we shall also have one
         assert_ne!(divisor, T::ZERO, "Cannot divide by zero");
 
+        assert!(
+            T::BITS >= numerator_bits as usize,
+            "The scalar divisor type must have a number of bits that is \
+            >= to the number of bits encrypted in the ciphertext"
```

Comment:
can we maybe give the current number of bits encrypted in the ciphertext and the bits we got for the scalar ?